Engine/Module/ModuleDebugDraw.cpp

```diff
@@ -0,0 +1,619 @@
+#include "ModuleDebugDraw.h"
+#include "Component/ComponentCamera.h"
+
+#define DEBUG_DRAW_IMPLEMENTATION
+#include "UI/DebugDraw.h"     // Debug Draw API. Notice that we need the DEBUG_DRAW_IMPLEMENTATION macro here!
+
+#include "GL/glew.h"
+#include <assert.h>
+
+class IDebugDrawOpenGLImplementation final : public dd::RenderInterface
```

Comment:
Maybe we can do the class declaration in the UI/DebugDraw.h  or create another header and define it in a different .cpp

Reply:
I tried the second approach but it generated conflicts with imports. I might need your help on this.